pkgs/tools/misc/grub/default.nix

```diff
@@ -42,8 +40,35 @@ let
   canEfi = lib.any (system: stdenv.hostPlatform.system == system) (lib.mapAttrsToList (name: _: name) efiSystemsBuild);
   inPCSystems = lib.any (system: stdenv.hostPlatform.system == system) (lib.mapAttrsToList (name: _: name) pcSystems);
 
-  version = "2.06";
+  gnulib = fetchFromSavannah {
+    repo = "gnulib";
+    # NOTE: keep in sync with bootstrap.conf!
+    rev = "9f48fb992a3d7e96610c4ce8be969cff2d61a01b";
+    hash = "sha256-mzbF66SNqcSlI+xmjpKpNMwzi13yEWoc1Fl7p4snTto=";
+  };
+
+  src = fetchFromSavannah {
+    repo = "grub";
+    rev = "6425c12cd77ad51ad24be84c092aefacf0875089";
+    hash = "sha256-PSCa993Reph6w9+leE4a/9E6vIALdOqU3FZEPwasFyk=";
+  };
 
+  # HACK: the translations are stored on a different server,
+  # not versioned and not included in the git repo, so fetch them
+  # and hope they don't change often
+  locales = runCommand "grub-locales" {
```

Comment:
would this work with runCommandNoCC?

Reply:
There is no `runCommandNoCC` anymore, `runCommand` is always `NoCC`.

Reply:
Oh, that's good to know!